### ISSUE TITLE
Breaks when attributes are wrapped in brackets

### ISSUE THREAD

**Issue by semaperepelitsa** [SublimeText/Slim#1]
No syntax highlighting in code following wrapped attributes:

```slim
div.example[data-attribute="foo"
            class=("current" if current?)]
  div.example
  div.example
  div.example
```

Sublime:
<img width="371" alt="Screenshot 2023-05-31 at 6 56 59 PM" src="https://github.com/SublimeText/Slim/assets/347921/6f796a89-2d3d-4277-82d9-374af7e25baa">

TextMate for comparison:
<img width="365" alt="Screenshot 2023-05-31 at 6 56 52 PM" src="https://github.com/SublimeText/Slim/assets/347921/11f2eff3-a2c5-4e2b-ac98-b34942359cde">

**Comment by deathaxe** [comment]
Thank's for your feedback. The issue should be fixed with v2.0.2

**Comment by semaperepelitsa** [comment]
Great, thanks for the quick fix!
